README.md

```diff
@@ -28,8 +28,27 @@ var MyComponent = onClickOutside(React.createClass({
 }));
 
 ```
+Note that if you try to wrap a React component class without `handleClickOutside(evt)` handler like this, the HOC will throw an error.
+If you want to execute method different than `handleClickOutside(evt)`, you can specify it as a second parameter to `onClickOutside` like this:
```

Comment:
"In order to use a custom event handler, you can specify the function to be used in the HOC options objects, passed as second parameter:"